cpp/example_code/s3/delete_bucket.cpp

```diff
@@ -51,8 +51,8 @@ bool AwsDoc::S3::DeleteBucket(const Aws::String& bucketName,
 int main()
 {
     //TODO: Change this bucket_name to the name of a bucket that actually exists in your account.
-    //If the bucket does not exist, you will get an "access denied" error.
-    Aws::String bucket_name = "my-bucket";
+    //Else, you will get either an Access Denied or a NoSuchBucket error (depending on whether the bucket exists elsewhere).

```

Comment:
If the bucket is not in your account, you will get one of two errors: AccessDenied if the bucket exists in some other account, or NoSuchBucket if the bucket does not exist in any account.